extensions/wikia/Email/styles/weeklyDigest.css

```diff
@@ -0,0 +1,24 @@
+h3 {
+	font-family: "Helvetica Neue", Arial, Sans-serif;
```

Comment:
Just FYI there's no need to put font names in quotes. Modern browsers don't care.